dark/reads.py

```diff
@@ -1512,6 +1512,30 @@ def variableSites(self, confirm=False, homogeneityLevel=1.0,
 
         return varSites
 
+    def combineReads(self):
+        """
+        Combine all reads into a single read.
+        """
+        reads = list(self)
+        length = len(reads[0].sequence)
```

Comment:
What if the reads in `self` have different lengths?

Reply:
You don't need the `.sequence` here. The `Reads` class has a `__len__` method that does that for you.